templates/profiles/change_password.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load markdownify %}
+{% load courses_extra %}
+{% load bootstrap_tags %}
+
+{% block content %}
+<form method="post">
+    {% csrf_token %}
+    <div class="py-5 container bg-light">
+            <table style="margin-right: auto; margin-left: auto">
+                {{ form.as_table }}
```

Comment:
Each form should have csrf token otherwise it is an security issue.

You can format form to bootstrap directly, since you have loaded `bootstrap_tags`:
 
```
<div class="col-3 mx-auto py-2">
        {% csrf_token %}
        {{ form|as_bootstrap }}
</div>
```